### PR TITLE
Add Github issue & PR templates

### PR DESCRIPTION
**Pull request by moorscode**
Based upon the ones in the WordPress (Yoast) SEO repository.

### CODE REVIEW

**Comment by kraftner** [comment]
Looks good.

Only question: Should it link to contribution guidelines on this repo as the one you linked to might confuse people as they actually are about another plugin?

**Comment by omarreiss** [comment]
@kraftner agreed. Moved this back to queue to be fixed.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!-- Please use this template when creating an issue. 
+- Please check the boxes after you've created your issue.
+- Please use the latest version of
+    ACF Content Analysis for Yoast SEO,
+    Yoast SEO
+    and Advanced Custom Fields.
+-->
+
+* [ ] I've read and understood the [contribution guidelines](https://github.com/Yoast/wordpress-seo/blob/trunk/.github/CONTRIBUTING.md).
```
Comment (omarreiss) [inline .github/ISSUE_TEMPLATE.md]: This should refer to a CONTIBRUTING.md in this repo.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+To receive free support for ACF Content Analisys for Yoast SEO we have the following channels:
+* [Support forums](https://wordpress.org/support/plugin/acf-content-analysis-for-yoast-seo/) on WordPress.org
+
+## Security issues
+Please do not report security issues here. Instead, email them to security at yoast dot com so we can deal with them securely and quickly.
+
+## I have found a bug
+Before opening a new issue, please:
+* update to the newest versions of WordPress and ACF Content Analisys for Yoast SEO.
```
Comment (kraftner) [inline .github/CONTRIBUTING.md]: Typo in "Analisys"

Also we should probably propose updating ACF and Yoast SEO as well. Or do we care about bugs with old versions that much?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Community made patches, localizations, bug reports and contributions are very welcome and ACF Content Analisys for Yoast SEO to become the best it can be.
+
+When contributing please ensure you follow the guidelines below so that we can keep on top of things.
+
+#### Submitting an issue you found
+Make sure your problem does not exist as a ticket already by searching through [the existing issues](https://github.com/Yoast/yoast-acf-analysis/issues). If you cannot find anything which resembles your problem, please [create a new issue](https://github.com/Yoast/yoast-acf-analysis/issues/new).
+
+#### Fixing an issue
+
+* Fork the repository on GitHub (make sure to use the trunk branch, not master).
```
Comment (kraftner) [inline .github/CONTRIBUTING.md]: We don't have `trunk` -> should probably be `develop`

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+* Make the changes to your forked repository.
+* Ensure you stick to the [WordPress Coding Standards](http://make.wordpress.org/core/handbook/coding-standards/) and you properly document any new functions, actions and filters following the [documentation standards](http://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/).
+* When committing, reference your issue and include a note about the fix.
+* Push the changes to your fork and submit a pull request to the 'trunk' branch of the Yoast SEO repository.
```
Comment (kraftner) [inline .github/CONTRIBUTING.md]: `develop` again

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+### How can we reproduce this behavior?
+1. 
+2. 
+3. 
+
+### Technical info
+* WordPress version: 
+* ACF Content Analysis for Yoast SEO version: 
+* Yoast SEO version: 
+* ACF version:
```
Comment (kraftner) [inline .github/ISSUE_TEMPLATE.md]: The free version will soon move to 5.* too so whether it is free or pro would be nice to mention. Maybe:

> ACF version (including if Free or Pro):

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (moorscode) [reply .github/CONTRIBUTING.md]: Good point, I'll add it.

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+## I want to create a patch
+Community made patches, localizations, bug reports and contributions are very welcome and ACF Content Analysis for Yoast SEO to become the best it can be.
+
+When contributing please ensure you follow the guidelines below so that we can keep on top of things.
+
+#### Submitting an issue you found
+Make sure your problem does not exist as a ticket already by searching through [the existing issues](https://github.com/Yoast/yoast-acf-analysis/issues). If you cannot find anything which resembles your problem, please [create a new issue](https://github.com/Yoast/yoast-acf-analysis/issues/new).
+
+#### Fixing an issue
+
```
Comment (andizer) [inline .github/CONTRIBUTING.md]: Is there a reason for this new line? This is the only place where a new line after a heading is added.

**Comment by jcomack** [comment]
Acceptance done 👍

**Pull request merged**